src/component/provider.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import ReduxStoreContext from '../context/redux-store'
+
+class Provider extends Component {
+  constructor(props) {
+    super(props)
+    const { store } = this.props
+    this.state = store
```

Comment:
`this.store = store`  or `this.state = store.getState()` ?

Reply:
I've added a commit to fix this

Reply:
I've added a commit to fix this